compass/landice/tests/greenland/high_res_mesh/high_res_mesh.cfg

```diff
@@ -0,0 +1,29 @@
+# config options for high_res_mesh test case
+[high_res_mesh]
```

Comment:
Yes, I thought about mentioning that earlier but decided to let it be.  But these sections should make clear if they are specific to a test group (or test case) or generic to any landice test case that needs `high_res_mesh` config options.

Reply:
To be clear, should I change just the section name to `high_res_GIS_mesh` and leave the test case name as `greenland/high_res_mesh`? Or should I change all instances of `high_res_mesh` in this test case to `high_res_GIS_mesh`?

Reply:
Changed the section name in https://github.com/MPAS-Dev/compass/pull/351/commits/03f0a464438e68a99d5e485c36a22951946dacc3, but left all others of `high_res_mesh` as they were. Let me know if I should change those too.

Reply:
I think that is correct.  @xylar , can you confirm that makes sense to you?

Reply:
Yes, that's fine.  There is no need for the section name in the config file to match the test-case name.  It just is helpful if the section name is unique somehow.  Naming the test case `high_res_GIS_mesh` would be redundant, since it's in the `greenland` test group.